CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.5
+
+<!-- generated comparing v5.2.4..master -->
+
+_Dec 20, 2021_
+
+A big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 🛠 This release mostly improves what's behind the scenes: infrastructure and tests
```

Comment:
```suggestion
- 🛠 Improve infrastructure and increase test coverage
```
I think the shorter the better for posting on twitter.

Reply:
I wanted to add a bit of variety to these cold statements ;) As for Twitter, we don't need to quote the release quotes exactly there.